internal/server/handlers.go

```diff
@@ -60,48 +60,27 @@ func (a *API) GetUser(c *gin.Context, r *api.GetUserRequest) (*api.User, error)
 func (a *API) CreateUser(c *gin.Context, r *api.CreateUserRequest) (*api.CreateUserResponse, error) {
 	user := &models.Identity{Name: r.Name}
 
-	setOTP := r.SetOneTimePassword
-
-	// infra identity creation should be attempted even if an identity is already known
```

Comment:
Removing this because the behaviour is not in line with other POST endpoints. Instead, the user should use the UpdateUser endpoint to create or update the password. A side effect of this change is that admins will no longer be able to (re)set an one-time password one the user exists

Reply:
Would it be possible to do this in a separate PR? 

Is the change to `AssignIdentityToGroups` related to this?

Reply:
I'll split into a separate PR. It's something I noticed while testing the changes. When creating duplicate users, the error return complains about duplicate credentials rather than duplicate users which was surprising